test/e2e/servers/graphql-main-server/with-auth-requests.spec.ts

```diff
@@ -336,6 +353,36 @@ describe("graphql", () => {
     })
   })
 
+  describe("lnUsdInvoiceCreate", () => {
+    const mutation = LN_USD_INVOICE_CREATE
+
+    it("returns a valid lightning invoice", async () => {
+      const input = {
+        walletId: usdWalletId,
+        amount: 1000,
+        memo: "This is a lightning invoice",
+      }
+      const result = await apolloClient.mutate({ mutation, variables: { input } })
+      const { invoice, errors } = result.data.lnUsdInvoiceCreate
+      expect(errors).toHaveLength(0)
+      expect(invoice).toHaveProperty("paymentRequest")
+      expect(invoice.paymentRequest.startsWith("lnbcrt4")).toBeTruthy()
+      expect(invoice).toHaveProperty("paymentHash")
+      expect(invoice).toHaveProperty("paymentSecret")
+    })
+
+    it("returns a valid lightning invoice if memo is not passed", async () => {
+      const input = { walletId: usdWalletId, amount: 1000 }
+      const result = await apolloClient.mutate({ mutation, variables: { input } })
+      const { invoice, errors } = result.data.lnUsdInvoiceCreate
+      expect(errors).toHaveLength(0)
+      expect(invoice).toHaveProperty("paymentRequest")
+      expect(invoice.paymentRequest.startsWith("lnbcrt4")).toBeTruthy()
+      expect(invoice).toHaveProperty("paymentHash")
+      expect(invoice).toHaveProperty("paymentSecret")
+    })
+  })
```

Comment:
this test doesn't seems to bring more value than the prior one, as we're testing the dealer integration here and the only difference should be in price/amount

Reply:
True. I just copy pasted the 2 tests from the non usd structure. Will delete it